config/environments/production.rb

```diff
@@ -56,6 +56,7 @@
   # config.active_job.queue_adapter     = :resque
   # config.active_job.queue_name_prefix = "rescue_rails_#{Rails.env}"
   config.action_mailer.perform_caching = false
+  config.action_mailer.delivery_method = :smtp
```

Comment:
Don't all our emails go through mailchimp?

Reply:
ah I see, from the mailhopper config

👍

Reply:
I don't know, Shaun. I don't know how mailchimp works. If someone has some history with the mailer I'd appreciate some guidance.

Reply:
I think you're good here, let's just make sure non mailchimp mails work still when we're on staging.

@markottaviani any idea which emails those are?

Reply:
how do you do staging for this app? Is there a staging server with public access somewhere?

Looks like mailchimp is used for User (un)subscribe and Adopter (un)subscribe (in app/jobs). Both the Adopter and User models have callbacks on create and update to trigger mailchimp emails.

Reply:
I note that the Travis-CI build is now failing again. It's the same timing issue related to sending emails that I just fixed on another  PR. I'll port the same fix into this PR. Please stand by...

Reply:
Build is passsing. Is there anything I need to do related to staging?

Reply:
Everything in `app/mailers`, "transactional emails" goes out via smtp & sendgrid.com .

RescueRails only concern with Mailchimp is adding, removing and updated the mailing lists in Mailchimp. 

I'll make sure the email plumbing still works the same in the staging env.